src/Sylius/Bundle/AttributeBundle/Form/EventSubscriber/BuildAttributeValueFormSubscriber.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Sylius\Bundle\AttributeBundle\Form\EventSubscriber;
 
+use Sylius\Bundle\AttributeBundle\Form\Type\AttributeValueType\Configuration\Factory\AttributeValueTypeConfigurationFactoryInterface;
```

Comment:
I think the whole thing should go to ``Sylius\Bundle\AttributeBundle\Factory\*``(or even ``Sylius\Component\Attribute\Factory\*``?) and then we can talk about renaming :smile: